pkg/adapter/adapter.go

```diff
@@ -52,8 +51,8 @@ type ExchangeConfig struct {
 }
 
 type ChannelConfig struct {
-	PrefetchCount int
-	GlobalQos     bool
+	PrefetchCount int  `envconfig:"RABBITMQ_CHANNEL_CONFIG_PREFETCH_COUNT" default:"10" required:"false"`
```

Comment:
This environment name feels too verbose and implementation-specific. I think that `RABBITMQ_PREFETCH_COUNT` is sufficiently descriptive - wdyt?

Since `PrefetchCount` and `GlobalQos` are both used in the context of `Qos`, I think that it would make sense to rename them to `QosPrefetchCount` and `QosGlobal`.

Reply:
same as in the globalQos